main.py

```diff
@@ -0,0 +1,52 @@
+import os
+import json
+
+from geolocalizer import Geolocalizer
+
+def hello_world(request):
+    """Responds to any HTTP request.
+    Args:
+        request (flask.Request): HTTP request object.
+    Returns:
+        The response text or any set of values that can be turned into a
+        Response object using
+        `make_response <http://flask.pocoo.org/docs/1.0/api/#flask.Flask.make_response>`.
+    """
+    request_json = request.get_json()
+    if request.args and 'message' in request.args:
+        return request.args.get('message')
+    elif request_json and 'message' in request_json:
+        return request_json['message']
+    else:
+        return f'Hello World!'
+
+def geo_localize_image_map(request, mock=False):
+    """
+    Args:
+        request (flask.Request): HTTP request object
+        Json example: {"uri": "https://i.stack.imgur.com/WiDpa.jpg"}
+    """
+    if mock:
+        uri = request['uri']
```

Comment:
I am not a fan of branching code for test purposes only. Cloud functions offers solutions to test a function locally without deploying. I think we should rely on those and avoid branching the code. Note that the request may get more complicated later, for example we may want to add {"country":"US"} if we are sure the map is within US. The current approach will double the efforts for further developments.

Reply:
Deleted, thanks.